src/wp-includes/navigation.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Hooks and utilities for Navigation in the Block Editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Preloads requests needed for Navigation posts
+ *
+ * @param array                   $preload_paths    Preload paths to be filtered.
```

Comment:
There is an indentation issue on the params :)

Reply:
Thanks for the review. 

I'm afraid I'm not following this feedback. The params seems to be aligned and the PHPCS CI tests on this PR pass. 

What needs to change here?

Reply:
There are 3 unwanted spaces after `$preload_paths` and `$context` :)